scripts/feature_tests/upgrade/Makefile

```diff
@@ -1,8 +1,8 @@
 M3PATH := "$(dirname "$(readlink -f "${0}")")/../../"
 # This needs to be moved to upgrade.sh and different value should be set before
 # each use case.
-# NUM_OF_MASTER_REPLICAS := 1
-# NUM_OF_WORKER_REPLICAS := 1
+# export NUM_OF_MASTER_REPLICAS := 1
+# export NUM_OF_WORKER_REPLICAS := 1
 
```

Comment:
upgrade tests modify these variables between and during(not all cases) test runs so the exports are not actually needed here

Reply:
OK, I added them so that you also don't run to the same issue in case if you use those vars. Those lines will be deleted anyway when upgrade is ready.